guides/source/debugging_rails_applications.md

```diff
@@ -242,6 +242,44 @@ The contents of the block, and therefore the string interpolation, is only
 evaluated if debug is enabled. This performance savings is only really
 noticeable with large amounts of logging, but it's a good practice to employ.
 
+
+Debugging with the Web Console
+-------------------------------
+
+The web console allows you to create an interactive ruby session in your browser. An interactive 
+console is launched automatically in case on an error but can also be launched for debugging purposes
+by invoking `console` in a view or controller.
+
+For example in a view:
+
+```ruby
+# new.html.erb
+<%= console %>
+```
+
+Or in a controller:
+
+```ruby
+# posts_controller.rb
+class PostsController < ApplicationController
+  def new
+    console
+    @post = Post.new
+  end
+end
+```
+
+By default the web console can only be accessed from localhost. `config.web_console.whitelisted_ips` 
+lets you control which IPs have access to the console. For example, to allow access from both localhost
+and 192.168.0.100:
+
+```ruby
+# config/application.rb
+class Application < Rails::Application
+  config.web_console.whitelisted_ips = %w( 127.0.0.1 192.168.0.100 )
```

Comment:
Maybe its worth noting that you can whitelist a whole network as well:

```ruby
class Application < Rails::Application
  config.web_console.whitelisted_ips = %w( 127.0.0.1 192.168.0.0/16 )
end
```
Also, that outside of the [private networks](http://en.wikipedia.org/wiki/Private_network), this is pretty insecure :)